apps/site/lib/site_web/views/fare_transformation_view.ex

```diff
@@ -3,4 +3,14 @@ defmodule SiteWeb.FareTransformationView do
   View for the Fare Transformation section of the website.
   """
   use SiteWeb, :view
+
+  alias GoogleMaps.Geocode.Address
```

Comment:
**thought(non-blocking):** This will change to `LocationService.Address` when merged in with the branch with the AWS work.